### ISSUE TITLE
add placeholder content for components that are loading

### ISSUE THREAD

**Issue by dwhieb** [digitallinguistics/app#142]
Display an accessible, visual placeholder for content that is waiting to be loaded. This placeholder should be consistent across components.
